publish/publish-type.proj

```diff
@@ -48,11 +48,9 @@
     <!-- TODO: We should also think about pulling down the checksums or recomputing them -->
     <PropertyGroup>
       <DownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(BinDir)PackageDownload/</DownloadDirectory>
-      <PublishDirectory Condition="'$(PublishDirectory)' == ''">$(BinDir)ForPublishing/</PublishDirectory>
-      <!-- if we're not signing packages, publish directly from the download directory, as we won't be
-	       copying them to the indexed directory -->
-      <PublishDirectory Condition="'$(WindowsSdkDir)' == '' OR '$(SignType)' != 'real'">$(DownloadDirectory)</PublishDirectory>
+      <PublishDirectory Condition="'$(PublishDirectory)' == ''">$(DownloadDirectory)</PublishDirectory>
```

Comment:
We should probably just eliminate PublishDirectory and always use DownloadDirectory. As it stands right now someone could accidentally set them independently which would break things.